### ISSUE TITLE
Set data source for recipient field

### ISSUE THREAD

**Issue by bartrylant** [verbb/formie#1421]
### What are you trying to do?

I have a form (or multiple forms) with multiple recipient fields. These fields all contain the same data, which changes a few times per year. It would be nice if I could set a data source (in this case: an entries channel) instead of having to copy-paste 100 addresses in every recipient field, and keep all those fields up to date when something changes.

### What's your proposed solution?

Add a channel as a data source for the recipient field.
See [screenshot](https://imgur.com/a/mqvYbNP)

**Comment by engram-design** [comment]
Appreciate the suggestion, but I don't think a channel of entries is the correct "source" to be used here (a entry per row in the recipients table options). We would also need to be pretty strict with the source value needing to be an email and string. We probably need something a bit more purpose-built to store default settings like this.

It's for this exact reason we recommend using a Synced field. Each time you edit the settings of the field, they're synchronised between all instances of the field.

**Comment by bartrylant** [comment]
I also considered using a synced field. The problem is I need multiple of the same field in one form (the selection from recipient field 1 needs another notification than the selection of recipient field 2). And I need to change the label and name for the field. On one form the first instance has the name "Old club" and the second instance "New club". On another form it should be labeled "Select your current club".

**Comment by engram-design** [comment]
Yeah, fair enough. having a "dynamic source" field is an interesting idea, although I'm not _quite_ sure how to pull it off in a way that makes sense for content editors where they aren't falling over themselves just trying to update values. Or, worse - they update values not realising that they're "global". So there's some things to consider...

**Comment by bartrylant** [comment]
Ok I see. Would it be an option to add a "add bulk options" button similar to what you have on a dropdown field? That way I can just paste in a list in this format:
```
email1@domain.com,This is the name
email2@otherdomain.com,This is another name
```

**Comment by engram-design** [comment]
Possibly a quick-fix option, but I'll continue to give this some thought on the best way to handle it.

**Comment by engram-design** [comment]
Updated in [4.0.0-beta.10](https://github.com/verbb/formie/releases/tag/4.0.0-beta.10).
